cc3200/appsign.sh

```diff
@@ -13,7 +13,12 @@ BTYPE=$2
 BUILD=build/${BOARD}/${BTYPE}
 
 # Generate the MD5 hash
+# md5 on Darwin, md5sum on Unix
+if [ `uname -s` = "Darwin" ]; then
+echo -n `md5 $BUILD/application.bin | awk '{ print $4 }'` > __md5hash.bin
```

Comment:
You can use `md5 -q $BUILD/application.bin` to get only the checksum, without using awk